test/utils/Address.test.js

```diff
@@ -355,28 +329,5 @@ contract('Address', function (accounts) {
         recipient,
       ]);
     });
-
-    it('bubbles up error if specified', async function () {
-      await expectRevertCustomError(
-        this.mockFnPointer.functionCallWithValue(this.target.address, '0x12345678', 0),
-        'CustomRevert',
-        [],
-      );
-    });
-  });
-
-  describe('verifyCallResult', function () {
-    it('returns returndata on success', async function () {
-      const returndata = '0x123abc';
-      expect(await this.mockFnPointer.verifyCallResult(true, returndata)).to.equal(returndata);
-    });
```

Comment:
I think we still have to keep this.

Reply:
Re added in [3d09549](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4502/commits/3d09549bd3cd5110f78979e217d7e2d4e9cf2434)